network/gossip/libp2p/libp2pNode_test.go

```diff
@@ -48,19 +46,18 @@ func TestLibP2PNodesTestSuite(t *testing.T) {
 }
 
 // SetupTests initiates the test setups prior to each test
-func (l *LibP2PNodeTestSuite) SetupTest() {
-	l.logger = log.Output(zerolog.ConsoleWriter{Out: os.Stderr}).With().Caller().Logger()
-	l.ctx, l.cancel = context.WithCancel(context.Background())
-	golog.SetAllLoggers(golog.LevelDebug)
```

Comment:
Yahya - I leave this in to enable libp2p logging. Othewise I keep forgetting how to turn on libp2p logging. We can keep this line but just set its level to Error as well. This line enables libp2p logging.

Reply:
fixed in https://github.com/onflow/flow-go/pull/99/commits/4ba7992a9f96a1946722504673ed013b2c223252.